audformat/core/table.py

```diff
@@ -891,14 +928,226 @@ def _load_pickled(self, path: str):
 
         self._df = df
 
+    def _pyarrow_convert_dtypes(
+        self,
+        df: pd.DataFrame,
+        *,
+        convert_all: bool = False,
+    ) -> pd.DataFrame:
+        r"""Convert dtypes that are not handled by pyarrow.
+
```

Comment:
```suggestion
```

causes D202: "No blank lines allowed after function docstring"

Reply:
Now, I'm getting:

```
D205 1 blank line required between summary line and description
```

Seems like our settings are not compatible?

Reply:
>  "No blank lines allowed after function docstring"

The docstring was also not finished at this line, just the first line of it. Maybe you put your suggestion on the wrong line?

Reply:
>> Maybe you put your suggestion on the wrong line?

Ah probably this is it. D205  you get is on blank lines inside the docstring. I had been targeting D202 which is on the blank line between docstring and first line of code of the function. Apologies.

I am using the project's `pyproject.toml`. Maybe the ruff version has diverged?

Reply:
> Maybe the ruff version has diverged?

If you use `pre-commit` to run `ruff`, this shouldn't be the case as it handles installing the correct `ruff` version, e.g.

```bash
$ pre-commit install
$ pre-commit run --all-files
ruff.....................................................................Passed
ruff-format..............................................................Passed
codespell................................................................Passed
```

Reply:
I know but sometimes this is impossible - in particular when `python3.10`  as configured in `.pre-commit-config.yaml` is locally not installable - as on my box at the moment.  I would then fall-back to a local ruff check.  I never had this problem, but in future I should probably match  the version that `pre-commit` prescribes.

Reply:
I guess you should ask for a new PC if you cannot install Python 3.10 ;)